### PR TITLE
docs: doc limitations

### PR DESCRIPTION
**Pull request by NathanFlurry**

### CODE REVIEW

**Comment by NathanFlurry** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1567?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1567** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1567?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1567?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#1558** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1558?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1551** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1551?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: docs/src/content/docs/limitations.mdx [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
+| ------------------------- | ----- |
+| **JavaScript Build Size** | 10 MB |
+| **Container Build Size**  | 8 GB  |
+
+## Actor & Build Tags
+
+| Feature       | Limit      |
+| ------------- | ---------- |
+| **Tags**      | 8 tags     |
+| **Tag Key**   | 16 bytes   |
```
Comment (MasterPtato) [inline docs/src/content/docs/limitations.mdx:37]: Keys are 256 https://github.com/rivet-gg/rivet/blob/main/packages/services/ds/src/workflows/server/mod.rs#L248
```suggestion
| **Tag Key**   | 256 bytes   |
```

**Review by MasterPtato — CHANGES_REQUESTED** [review]
comment

File: docs/src/content/docs/limitations.mdx (hunk 1 above)
Comment (NathanFlurry) [reply docs/src/content/docs/limitations.mdx:37]: See https://linear.app/rivet-gg/issue/RVT-4353/lower-tag-limits-both-actors-and-builds-to-8

**Review by MasterPtato — APPROVED** [review]

**Pull request closed without merging**